### PR TITLE
PIMS PSP v16.03

### PR DESCRIPTION
**Pull request by dfilteau**
PSP_PIMS S16.03 Sprint 16 2021-Dec-08
- Altered seed DML file:
  - 45_DML_PIMS_COUNTRY.sql
    - Added display order for country.
- Requires additional metadata to meet standards

### CODE REVIEW

File: database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+GO
+
+CREATE VIEW PIMS_CONTACT_MGR_VW AS
+  SELECT CONCAT('P', PER.PERSON_ID) AS ID
+       , PER.PERSON_ID
+       , NULL                   AS ORGANIZATION_ID
+       , PER.IS_DISABLED
+       , TRIM(CONCAT_WS(' ', PER.FIRST_NAME, PER.MIDDLE_NAMES, PER.SURNAME)) AS SUMMARY
+       , PER.SURNAME
+       , PER.FIRST_NAME
```
Comment (asanchezr) [inline database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql:17]: Not sure this change belongs here? the changelog below only lists changes to the PIMS_COUNTRY ordering...

Comment (asanchezr) [reply database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql:17]: Also the file name seems wrong - `..._16.01.sql` doesn't seem to indicate it should be applied in 16.03

Comment (asanchezr) [reply database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql:17]: ^^ @dfilteau ?

Comment (dfilteau) [reply database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql:17]: I think the 16.01 PR has 16.03 added onto it.

Comment (asanchezr) [reply database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql:17]: Any change you could delete this file from the PR? It is already under folder **S16_02** Alter Up so no sense in having it for folder **S16_03**

Comment (asanchezr) [reply database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql:17]: ^ @dfilteau

Comment (dfilteau) [reply database/mssql/scripts/dbscripts/PSP_PIMS_S16_03/Alter Up/00 PIMS_CONTACT_MGR_VW Alter_v16.01.sql:17]: Yup.  Can do.

**Comment by asanchezr** [comment]
Closing this PR. Changes will be rolled into S17

**Pull request closed without merging**